runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayManager.java

```diff
@@ -113,26 +113,23 @@ public void onPluginChanged(final PluginChanged event)
 	@Subscribe
 	public void onMenuOptionClicked(MenuOptionClicked event)
 	{
-		if (event.getMenuAction().equals(MenuAction.RUNELITE_OVERLAY))
+		if (MenuAction.RUNELITE_OVERLAY.equals(event.getMenuAction()))
```

Comment:
by flipping the if I meant if (!blahblah) return, lol

Reply:
🤦‍♂️ Fixed now